source/_statement.html.md

```diff
@@ -33,14 +33,28 @@ The content listed below is non-accessible for the following reasons.
 
 ### Non compliance with the accessibility regulations
 
-The side navigation on the administrative platform sometimes does not accurately show where a user is on the website. This fails WCAG 2.1 AA success criterion 2.4.7 (focus visible).
+We have identified the following issues:
 
-Some menu entries do not appear in an active state when a user browses their content. This fails WCAG 2.1 AA success criterion 3.2.3 (consistent navigation).
+On our administrative portal, not all of our pages are identified with a unique title.
 
-We'll make all these elements fully accessible by September 2020.
+We'll make all these elements fully accessible by the end of the year.
+
+The following content on the GovWifi Status Page is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances, are listed below:
+
+* an incorrect heading hierarchy which fails WCAG 2.1A 1.3.1 success criterion (Info and Relationships).
```

Comment:
The formatting of the bullet points and punctuation here isn't quite right. 

As there's a lead in sentence, all the bullet points should start with a lower case letter, only contain one sentence, and not end in a full stop. This can confuse screen readers, because it's not sure whether to treat the content as bullet points or paragraphs. The [PaaS statement](https://www.cloud.service.gov.uk/accessibility-statement/) doesn't have a lead in line, so the bullet points are all self-contained and consistent. So you could do it either way.